transports/http/src/main/java/org/mule/transport/http/config/HttpsNamespaceHandler.java

```diff
@@ -23,24 +22,24 @@
 /**
  * Reigsters a Bean Definition Parser for handling <code><https:connector></code> elements.
  */
-public class HttpsNamespaceHandler extends AbstractMuleNamespaceHandler
+public class HttpsNamespaceHandler extends HttpNamespaceHandler
```

Comment:
I don't think you can just change the superclass like this without any side effects. Won't HttpNamespaceHandler register other beanProcessors?